packages/terra-consumer-layout/src/Layout.scss

```diff
@@ -28,7 +28,7 @@
     min-width: var(--terra-consumer-nav-width, 320px);
     position: relative;
     width: var(--terra-consumer-nav-width, 320px);
-    z-index: 100;
+    z-index: 200;
```

Comment:
This works because its likely you were actually clicking the overlay which had a matching [z-index of 100](https://github.com/cerner/terra-core/blob/master/packages/terra-overlay/src/Overlay.scss#L15).  IMO, overlay should have a higher z-index unless @bjankord has a reason for it being at 100. For now this fix is OK IMO